Mathlib/LinearAlgebra/AffineSpace/Basis.lean

```diff
@@ -275,6 +273,46 @@ instance instAddAction : AddAction V (AffineBasis ι k P) :=
   congr! 1
   rw [vadd_vsub_assoc, neg_add_eq_sub, vsub_vadd_eq_vsub_sub]
 
+section SMul
+variable [Group G] [Group G']
+variable [DistribMulAction G V] [DistribMulAction G' V]
+variable [SMulCommClass G k V] [SMulCommClass G' k V]
+
+/-- In an affine space that is also a vector space, an `AffineBasis` can be scaled.
+
+TODO: generalize to include `SMul (P ≃ᵃ[k] P) (AffineBasis ι k P)`, which acts on `V` with a `VAdd`
+version of a `DistribMulAction`. -/
```

Comment:
I added a TODO here, but I guess it applies to all of these actions?

Reply:
Yes, I think so

Reply:
Could you copy the `TODO` to the other new instances, where appropriate?

Reply:
I think I did that correctly, but it's 3am here so you might want to double check